driver/makedepend.ml

```diff
@@ -574,10 +577,11 @@ let print_version_num () =
 
 let main () =
   Clflags.classic := false;
-  add_to_list first_include_dirs Filename.current_dir_name;
   Compenv.readenv ppf Before_args;
   Clflags.reset_arguments (); (* reset arguments from ocamlc/ocamlopt *)
   Clflags.add_arguments __LOC__ [
+     "-nocwd", Arg.Set nocwd,
+        " Do not include current working dir in list of include directories";
```

Comment:
A last nitpick: the option should probably appear after `-I` since it is a negative option related to the major `-I` option.